examples/README.md

```diff
@@ -70,3 +70,26 @@ Then cd in the example folder of your choice and run
 ```bash
 pip install -r requirements.txt
 ```
+
+# FAQ
+
+Including a FAQ section could be useful for addressing common issues and solutions that users may encounter. This helps users deal with the errors they encounter more quickly and efficiently.
+
+1. **Question:** How do I install the `requirements.txt` file?
+   **Answer:** In the command line, you can run the following command: `pip install -r requirements.txt`. This installs all the packages listed in the requirements.txt file.
```

Comment:
We don't have a requirements.txt file in root actually but just in each example folder? Did you mean that?